app/src/main/java/com/hedvig/app/feature/referrals/ui/tab/ReferralsViewModel.kt

```diff
@@ -79,22 +74,10 @@ class ReferralsViewModelImpl(
         }
     }
 
-    private suspend fun createTopBar(): ViewState.Success.TopBarState? {
+    private suspend fun shouldShowCampaignBar(): Boolean {
+        val remoteConfig = RemoteConfig()
```

Comment:
Can we pass the RemoteConfig as a parameter to the ViewModel so that we don't have to do this every time?